Comparison Engine/Comparison Engine/Data/Data.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Comparison_Engine.Base_Classes;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Text.Json.Serialization;
+
+namespace Comparison_Engine
+{
+    class Data
+    {
+
+        public static List<Bar> GetBars()
+        {
+            StreamReader r = new StreamReader("barInput.json");
+            string json = r.ReadToEnd();
+            List<Bar> bars = JsonConvert.DeserializeObject<List<Bar>>(json);
+            return bars;
+        }
+
+        public static List<Bar> GetDrinks()
+        {
+            StreamReader r = new StreamReader("drinkInput.json");
+            string json = r.ReadToEnd();
+            List<Bar> drinks = JsonConvert.DeserializeObject<List<Bar>>(json);
+            return drinks;
+        }
```

Comment:
Ar šitam metode neturėtų būti List<Drink>? Also might need to add some form of validation or catch exceptions in the future for reading from file